apps/editing-toolkit/editing-toolkit-plugin/block-patterns/patterns/call-to-action-02.php

```diff
@@ -14,16 +14,16 @@
          <!-- wp:jetpack/layout-grid-column -->
          <div class="wp-block-jetpack-layout-grid-column wp-block-jetpack-layout-grid__padding-none">
             <!-- wp:paragraph {"align":"center","customTextColor":"#ffffff","style":{"typography":{"fontSize":80}},"className":"margin-bottom-none"} -->
-            <p style="color:#ffffff;font-size:80px" class="has-text-color has-text-align-center margin-bottom-none"><strong>' . esc_html__( 'Get it delivered', 'full-site-editing' ) . '</strong></p>
+            <p class="has-text-align-center" style="font-size:80px"><p style="color:#ffffff;font-size:80px" class="has-text-color has-text-align-center margin-bottom-none"><strong>' . esc_html__( 'Get it delivered', 'full-site-editing' ) . '</strong></p></p>
```

Comment:
Is this nested `<p>` intentional? That will cause problems going through supports code probably.

Reply:
Yeah, this one's a bit funky. I flagged them in Slack (p1599313110366800-slack-C015AL3QL7M); my hunch is that the paragraph block deprecation isn't working quite right here :slightly_frowning_face: 

![image (11)](https://user-images.githubusercontent.com/96308/92391429-fcbf5f80-f11c-11ea-8929-2e962df2c9bb.png)

Reply:
I'll try a manual fix for now.